Makefile

```diff
@@ -54,20 +54,15 @@ jetstream_config:
 	curl -LJ -o app/experimenter/outcomes/jetstream-config.zip https://github.com/mozilla/jetstream-config/archive/main.zip
 	unzip -o -d app/experimenter/outcomes app/experimenter/outcomes/jetstream-config.zip
 
-build_dev: jetstream_config
-	docker build --target dev -f app/Dockerfile -t app:dev app/
+build_dev: jetstream_config ssl
```

Comment:
Why are we adding SSL? Just curious

Reply:
Oh I had a `compose_build` step that would rebuild all the compose containers, and that would check for the ssl certs, but I realized we don't actually need the `compose_build` step at all since the only container we need to worry about rebuilding is the experimenter container, so I just took the `compose_build` out and lifted the `ssl` up here.